CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v0.118.0 [2023-09-07]
+
+_What's new?_
+
+- New tables added
+  - [aws_workspaces_directory](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_workspaces_directory) ([#1884](https://github.com/turbot/steampipe-plugin-aws/pull/1884))
+
+_Enhancements_
+
+- Added example query in `aws_ec2_instance` table doc for fetching subnet details of the instances. ([#1883](https://github.com/turbot/steampipe-plugin-aws/pull/1883)) (Thanks [@Pankaj-SinghR](https://github.com/Pankaj-SinghR) for the contribution!)
+
+_Bug fixes_
+
+- Fixed the column type of `sms_configuration_failure` column in `aws_cognito_user_pool` table. ([#1890](https://github.com/turbot/steampipe-plugin-aws/pull/1890)) (Thanks [@KTamas](https://github.com/KTamas) for the contribution!)
```

Comment:
```suggestion
- Fixed the data type of `sms_configuration_failure` column in `aws_cognito_user_pool` table to be of `STRING` type instead of `JSON`. ([#1890](https://github.com/turbot/steampipe-plugin-aws/pull/1890)) (Thanks [@KTamas](https://github.com/KTamas) for the contribution!)
```